libs/types/src/tokens.rs

```diff
@@ -452,7 +452,10 @@ pub mod usdc {
 	pub const CURRENCY_ID_LP_ETH_GOERLI: CurrencyId = CurrencyId::ForeignAsset(100_001);
 	pub const CURRENCY_ID_LP_BASE: CurrencyId = CurrencyId::ForeignAsset(100_002);
 	pub const CURRENCY_ID_LP_ARB: CurrencyId = CurrencyId::ForeignAsset(100_003);
-	pub const CURRENCY_ID_LP_CELO: CurrencyId = CurrencyId::ForeignAsset(100_004);
+
+	pub const CURRENCY_ID_LP_CELO_WORMHOLE: CurrencyId = CurrencyId::ForeignAsset(100_004);
+	pub const CURRENCY_ID_LP_CELO: CurrencyId = CurrencyId::ForeignAsset(100_006);
```

Comment:
`CURRENCYT_ID_LP_CELO` was `100_004`, and now it is `100_006`. I guess this is the intention of the migration, so wherever we used `CURRENCY_ID_LP_CELO`, now it's `CURRENCY_ID_LP_CELO_WORKHOLE`.

Reply:
Exactly!